src/index.js

```diff
@@ -83,7 +83,7 @@ loader.pitch = function(request) {
 				// only process entry exports
 				if (current.resource!==entry) return;
 
-				let exports = compilation.__workerizeExports || (compilation.__workerizeExports = {});
+				let exports = CACHE[entry] || (CACHE[entry] = {});
```

Comment:
The core issue is here. The `__workerizeExports` property does not seem to be persisted across re-compiles, resulting in an empty methods list. I propose using the CACHE object instead, since it seems to be used for this purpose anyway.

I assume the compilation object is somehow instanced by entry file, so I wanted to replicate that in the CACHE object if there are many workers in the project.

I think this will still require the end-user to restart the build when method names change, but that's a lot better than having to restart after any change.